### PR TITLE
AP-3 added Bond flavour and basic abstraction for control center

### PR DESCRIPTION
**Pull request by ravjit-cliqz**

### CODE REVIEW

**Comment by moaz-cliqz** [comment]
@ravjit-cliqz should we name it paid or premium?

File: Jenkinsfile.nightly [hunk 1]
```diff
@@ -260,6 +278,18 @@ stage('Send email') {
...
     )
+    bodytxt = """
+        ${projectName} - Build # ${env.BUILD_NUMBER}
+        Check console output at ${env.JOB_URL}${env.BUILD_NUMBER}/ to view the results.
+        Bond apk Uploaded to: http://repository.cliqz.com.s3.amazonaws.com/dist/android/nightly/bond/latest.apk
+        Bond X86 apk Uploaded to: http://repository.cliqz.com.s3.amazonaws.com/dist/android/nightly/bond/latest_x86.apk
+        ${changesSinceLastSuccess}
+        Issues Transtioned ${transitionIssueList}""".stripIndent()
+    emailext(
+        to: 'javier@cliqz.com stefano@cliqz.com ravjit@cliqz.com dominik.s@cliqz.com alina@cliqz.com inci@cliqz.com richard@cliqz.com moaz@cliqz.com oleksandra@cliqz.com',
```
Comment (spacifici) [inline Jenkinsfile.nightly]: Do not hardcode personal addresses in a Jenkins job, we can always create another mailing list

File: Jenkinsfile.nightly [hunk 2]
```diff
@@ -260,6 +278,18 @@ stage('Send email') {
         subject: '$PROJECT_NAME - Build # $BUILD_NUMBER',
         body: bodytxt
     )
+    bodytxt = """
```
Comment (spacifici) [inline Jenkinsfile.nightly]: This is overwriting the regular build mail.

File: mozilla-release/mobile/android/app/build.gradle [hunk 3]
```diff
@@ -215,6 +229,33 @@ android {
...
+        //photon should contain some implementation of CC. We take the implementation from standard version
+        photon {
+            java {
+                srcDir "${topsrcdir}/mobile/android/standard/java"
+            }
+        }
+        bond {
+            java {
+                srcDir "${topsrcdir}/mobile/android/bond/java"
+                srcDir "${topsrcdir}/mobile/android/app/src/photon/java" //need to include the sources in the photon folder
```
Comment (spacifici) [inline mozilla-release/mobile/android/app/build.gradle]: Why is the standard folder not included?

**Comment by spacifici** [comment]
Merged manually

**Pull request closed without merging**